pkg/apis/hawtio/v1alpha1/hawtio_types.go

```diff
@@ -63,6 +63,10 @@ type HawtioSpec struct {
 	// Note that the operator will recreate the route if the field is emptied,
 	// so that the host is re-generated.
 	RouteHostName string `json:"routeHostName,omitempty"`
+	// Name of the TLS secret with the custom certificate used for the route TLS termination
+	RouteTLSSecret corev1.LocalObjectReference `json:"routeTLSSecret,omitempty"`
+	// Name of the optional TLS secret with the CA certificate used for the route TLS termination
+	RouteTLSCaCertSecret corev1.LocalObjectReference `json:"routeTLSCaCertSecret,omitempty"`
```

Comment:
Can this be grouped into a dedicated `route` or `tls` struct like for the other spec fields?